starter-code/scripts/articleView.js

```diff
@@ -65,6 +65,40 @@ articleView.setTeasers = function() {
   });
 };
 
+articleView.initNewArticlePage = function() {
+  $('#export-field').hide();
+  $('#article-json').on('focus', function() {
+    $(this).select();
+  });
+  $('#new-form').on('change', articleView.create);
+};
+
+articleView.create = function() {
+  $('#article-preview').empty().fadeIn();
+
+  var formArticle = new Article({
+    title: $('#article-title').val(),
+    body: $('#article-body').val(),
+    author: $('#article-author').val(),
+    authorUrl: $('#article-author-url').val(),
+    category: $('#article-category').val(),
+    publishedOn: $('#article-published:checked').length ? new Date() : null
+  });
+
+  $('#article-preview').append(formArticle.toHtml('#article-template'));
+
+  $('pre code').each(function(index, block) {
+    hljs.highlightBlock(block);
+  });
+
+  $('#export-field').show();
+  $('#article-json').val(JSON.stringify(formArticle));
```

Comment:
You don't have to stringify this article object. I thought that would generate an error per our conversation last Friday, but it does not.